fe/fe-core/src/main/java/com/starrocks/catalog/Database.java

```diff
@@ -654,14 +655,14 @@ public String getCatalogName() {
         return catalogName;
     }
 
-    public synchronized void addFunction(Function function) throws UserException {
-        addFunctionImpl(function, false);
+    public synchronized void addFunction(Function function, boolean allowExists) throws UserException {
```

Comment:
can you override this function with default value as fasle?

Reply:
sure

Reply:
although... there's only one usage of this function in the src code. Is this a matter of making the calls in the test more concise?

Reply:
done

Reply:
@dirtysalt